apps/accounts/end_user_signup_forms.py

```diff
@@ -165,9 +165,11 @@ def save(self):
                                    )
 
         # TODO: Add Crosswalk Create.
+        # unhex the salt before use
+        unsalt = binascii.unhexlify(settings.USER_ID_SALT)
```

Comment:
What does this do to Sandbox? If we're changing this behavior, won't this break the Sandbox environment. And, in a similar vein, why isn't this issue ocurring on Sandbox?